.rubocop_todo.yml

```diff
@@ -523,18 +540,16 @@ Style/InverseMethods:
     - 'lib/tasks/jason.rake'
     - 'test/models/api_test.rb'
 
-# Offense count: 2
-Style/MethodMissingSuper:
-  Exclude:
-    - 'script/update_googlebots.rb'
-    - 'script/update_ip_stats.rb'
-
 # Offense count: 2
 Style/MissingRespondToMissing:
   Exclude:
     - 'script/update_googlebots.rb'
     - 'script/update_ip_stats.rb'
 
+# Offense count: 84
+Style/OptionalBooleanParameter:
+  Enabled: false
```

Comment:
There are too many methods (84) -- plus all method calls -- to deal with in this PR. So I created a separate story to deal with these. https://www.pivotaltracker.com/story/show/174567601